src/pages/Compare/utils.tsx

```diff
@@ -12,11 +12,13 @@ type ForwardChangeMap = {
   [line: string]: ChangeInfo[];
 };
 
-const mergeChanges = (
+const mergeChangesInPlace = (
```

Comment:
I'm trying to understand how this is "merging". It seems like it's just appending a change to a array of changes. This is probably fine in terms of what it does, but the name is confusing to me, as I assumed merging meant possibly dealing with duplicates as well.